Ladybird/Utilities.cpp

```diff
@@ -36,11 +32,11 @@ void platform_init()
 #ifdef AK_OS_MACOS
         return LexicalPath(app_dir).parent().append("Resources"sv).string();
 #else
-        return LexicalPath(app_dir).parent().append("share"sv).string();
+        return LexicalPath(app_dir).parent().append("share/ladybird"sv).string();
```

Comment:
I do wonder if share/lagom would be better? Then we could share it with MacPDF/LinuxPDF etc. This has the effect of putting some files in share/ladybird/ladybird, which feels strange :sweat_smile: